### DISCUSSION TITLE
How to insert special characters in titles of articles?

### DISCUSSION THREAD

**Discussion by earthmoon** [retorquere/zotero-better-bibtex#2379] [Q&A]
Hello,
I input special characters in the titles of the articles, for example:

>  Determination of spatiotemporal variability of tree water uptake using stable isotopes (δ18O, δ2H) in an alluvial system supplied by a high-altitude watershed, Pfyn forest, Switzerland

The **δ18O** and **δ2H** should be converted to **\chemdelta\isotope\*{18,O}** and  **\chemdelta\isotope\*{2,H}** in LaTeX, so I did as this in the field of the titles in the Zotero, but Better Bibter will convert them to **\textbackslash chemdelta\textbackslash isotope\*\{2,\vphantom\}{{H}}\vphantom\{\}** and other things like this. How should I input them correctly with Better Bibtex?

Thanks.

**Comment by earthmoon** [comment]
Okay. I asked a silly question again. I know now I should use **<script>**. Thanks a lot.

**Reply by earthmoon** [reply]
But I don't think `<script>` is a good idea. It is better to set one that doesn't convert any characters in the fields in the setting of the Better Bibtex.

**Reply by retorquere** [reply]
Sorry, I don't understand what you are suggesting here.

**Reply by earthmoon** [reply]
Let me express it in detail.
Using `<script>` is a little clumsy when there are too many references needed to be modified to insert `<script>`.  What we care about are the special characters in the title or other fields. For example, if we insert `\big{ }` in a title, Better Bibtex will convert it to another form to make it print as `\big{ }` in the paper, but sometime we actually may need is to run `\big{ }` in Latex file to make something big. So, how about adding a switch in the setting of the Better Bibtex tab of the references of Zotero. When the switch is on, Better Bibtex will make `\big{ }` printed as `\big{ }` characters. When the switch is off, Better Bitex will make `\big{ }` as Latex code to be run.
Do I express my idea clearly?
Thanks.

**Reply by retorquere** [reply]
https://retorque.re/zotero-better-bibtex/installation/preferences/hidden-preferences/#rawlatag

**Reply by earthmoon** [reply]
Yes! You do better. I think I have to read all words from https://retorque.re/zotero-better-bibtex/ carefully before asking a new question again. Thanks.

**Reply by retorquere** [reply]
No worries, it's always OK to ask. The docs for BBT are imperfect, and there's a lot to digest.

**Reply by earthmoon** [reply]
Thanks for your patience, tolerance and understanding. You do better than my imagination for Better Bibtex. As for my poor English, sometimes I don't digest the docs well.
